custom_components/tuya_local/devices/pc311ty_energy_meter.yaml

```diff
@@ -2,18 +2,6 @@ name: PC311-TY power clamp
 products:
   - id: vfwxetkavqrvg02u
 primary_entity:
-  entity: sensor
-  category: diagnostic
-  class: power
-  name: Total active power
-  dps:
-    - id: 9
-      type: integer
-      name: sensor
-      unit: W
-      force: true
-      class: measurement
-secondary_entities:
   - entity: sensor
```

Comment:
You'll need to remove the `-` here, as primary entity is not a list.

Reply:
I think this is good, sorry I am beginner to home assistant